theano/tensor/tests/test_blas.py

```diff
@@ -1335,15 +1315,16 @@ def get_data(self, x_stride=1, y_stride=1):
 
     def test_simple(self):
         alpha, beta, a, x, y = [self.shared(value)
-             for value in self.get_data()]
-        desired_oy = alpha.get_value() * matrixmultiply(a.
-            get_value(), x.get_value()) + beta.get_value() * y.get_value()
+                                for value in self.get_data()]
+        desired_oy = alpha.get_value() * \
+            matrixmultiply(a.get_value(), x.get_value()) + \
+            beta.get_value() * y.get_value()
```

Comment:
We generally prefer enclosing the expression in parentheses, rather than escaping the end-of-line, but it's OK.